media/CircleEditor/type.js

```diff
@@ -15,7 +15,7 @@
  */
 
 /* eslint-disable */
-let typeName = {
+var typeName = {
```

Comment:
I think `typeName` should be modified too because it is not matched well T.T
These are not typename but map whose key is `OptionName` and value is `OptionValue`.

Could you change this name to `optionValues`?
```suggestion
var optionValues = {
```